boards/xtensa/m5stack_atoms3/m5stack_atoms3_defconfig

```diff
@@ -0,0 +1,18 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_BOARD_M5STACK_ATOMS3=y
+CONFIG_SOC_SERIES_ESP32S3=y
+
+CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_ESP_HEAP_MEM_POOL_REGION_1_SIZE=0
```

Comment:
This is not needed nor available for ESP32S3. Only for ESP32 that has dual ram bank. So you can remove it from here.